### DISCUSSION TITLE
Appium version for node version v18.12.1 and npm version 8.19.2

### DISCUSSION THREAD

**Discussion by hasna9** [appium/appium#19548] [Q&A]
Hi, which Appium version is compatible with node version v18.12.1 and npm version 8.19.2?
because I downloaded the latest appium version 2.2.3, but some of its drivers are missing I guess when installed with node version v18.12.1 and npm version 8.19.2

**Comment by KazuCocoa** [comment]
> some of its drivers are missing 

What do you mean? In Appium v2, each driver you want should download individually
e.g. https://appium.io/docs/en/2.0/quickstart/uiauto2-driver/
